exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "detects reverse-string",
+      "property": "reverse-string",
+      "input": "robot",
+      "expected": "tobor"
+    },
+    {
+      "description": "detects empty string",
+      "property": "reverse-string",
+      "input": "",
+      "expected": "String is empty, input valid string"
+    },
+        {
+      "description": "detects reverse-string twice and check if it's original string",
+      "property": "reverse-string",
+      "input": "robot",
+      "expected": "robot"
```

Comment:
nothing other than the description tells us that reverse is supposed to be applied to this string twice. If it is to be, the `property` must be different.